pontoon/base/static/css/style.css

```diff
@@ -489,34 +498,49 @@ label {
   width: 219px; /* Instead of 220px; because of padding */
 }
 
-.locale.select {
-  padding-left: 0;
-  width: 100%;
+.part .menu nav ul li {
+  border: 1px solid transparent;
+  border-bottom-color: #5E6475;
 }
 
-.locale.select .menu {
-  bottom: auto;
-  display: inline-block;
-  position: relative;
-  width: 100%;
+.part .menu nav ul li.hover {
+  background: transparent;
+}
 
-  -moz-box-sizing: border-box;
-  box-sizing: border-box;
+.part .menu nav ul li.active {
```

Comment:
Selectors like this make it really hard to understand what this is actually styling. The class names are very brief, and it uses tag selectors, which generally are a bad idea because they make it hard to make changes to the HTML and are hard for a developer to remember VS a descriptive class name.

Granted, I've been searching for years for a good guide to CSS class naming because I've been disappointed with how I do it and haven't found it yet, so maybe it's impossible. :P